plugins/ata/ata.quirk

```diff
@@ -1,3 +1,7 @@
 # match all devices with this udev subsystem
 [DeviceInstanceId=BLOCK]
 Plugin = ata
+
+[DeviceInstanceId=OUI\002538]
+Vendor = Samsung
+VendorId = ATA:0x144D
```

Comment:
Here's another one that shows up in daemon logs from one of my workstations.

```
[DeviceInstanceId=OUI\000c50]
Vendor = Seagate
VendorId = ATA:0x1BB1
```

Reply:
Added, thanks.